std_msgs/Header header
float64 ctl_force_high                # ctl_force_high reading either
float64 ctl_force_high_variance       # 0 is interpreted as unknown


